src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataResourceSerializer.cs

```diff
@@ -634,6 +638,105 @@ public virtual ODataResource CreateResource(SelectExpandNode selectExpandNode, R
             }
         }
 
+        /// <summary>
+        /// AppendInstanceAnnotations
+        /// </summary>
+        /// <param name="resource"></param>
+        /// <param name="resourceContext"></param>
```

Comment:
add more descriptive summary and parameter descriptions.

Reply:
#Resolved

Reply:
Still not very descriptive. See AddDynamicProperties for descriptions that are at least a bit more descriptive.

---
In reply to: [458443062](https://github.com/OData/WebApi/pull/2219#discussion_r458443062) [](ancestors = 458443062)

Reply:
Sorry; I was okay with the description of the method, but not of the parameters.  i.e., looking for something like:
  /// <param name="resource">The <see cref="ODataResource"/> describing the resource being annotated.</param>
  /// <param name="resourceContext">The context for the resource instance being annotated.</param>

---
In reply to: [487202212](https://github.com/OData/WebApi/pull/2219#discussion_r487202212) [](ancestors = 487202212,458443062)